code/modules/mob/living/carbon/human/species/station/unathi_subspecies.dm

```diff
@@ -0,0 +1,126 @@
+/datum/species/unathi/autakh
+	name = "Aut'akh Unathi"
+	short_name = "aut"
+	name_plural = "Aut'akh Unathi"
+
+	icobase = 'icons/mob/human_races/r_autakh.dmi'
+	deform = 'icons/mob/human_races/r_def_lizard.dmi'
+	tail = "autakh"
+	tail_animation = null
+
+	slowdown = 0.5
+	brute_mod = 0.7
+	burn_mod = 1.2
```

Comment:
Their actual brute mod and burn mod values will be 0.63 and 1.32 respectively, since robotic limbs already have their own damage resists on top of this.

Reply:
yikes

Reply:
I will just remove the robotize() proc changing the limb resists, it always gets in the way of species mods.

Reply:
Done, burn and brute mods are now controlled by the robotic limb datum instead of being universal.